### PR TITLE
Create a route and View for Our Team Page

### PR DESCRIPTION
**Pull request by gauthamzz**
#565 is solved @aka-jain please review

### CODE REVIEW

File: frontend/src/views/web/partials/dashboard-footer.html [hunk 1]
```diff
@@ -18,6 +18,7 @@ <h5 class="white-text">Links</h5>
                     <li><a class="text-light-gray" href="#!">Blog</a></li>
                     <li><a class="text-light-gray" ui-sref="about-us">About Us</a></li>
                     <li><a class="text-light-gray" href="#!">Contact Us</a></li>
+                    <li><a class="text-light-gray" href="our-team">Our Team</a></li>
```
Comment (aka-jain) [inline frontend/src/views/web/partials/dashboard-footer.html]: @gauthamzz make it ui-sref please.

Comment (gauthamzz) [inline frontend/src/views/web/partials/dashboard-footer.html]: @aka-jain sorry, fixed it . is it okay now

**Comment by aka-jain** [comment]
@gauthamzz please resolve the conflicts.

**Comment by gauthamzz** [comment]
@aka-jain merge conflicts are solved :smile:

**Comment by aka-jain** [comment]
thanks @gauthamzz  :)

**Pull request merged**